connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/bop/SendEmailsRequest.java

```diff
@@ -1,19 +1,26 @@
 package com.redhat.cloud.notifications.connector.email.model.bop;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
 
 import java.util.HashSet;
 import java.util.Set;
 
 import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
 
 @JsonAutoDetect(fieldVisibility = ANY)
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class SendEmailsRequest {
 
     private final Set<Email> emails = new HashSet<>();
-    private final boolean skipUsersResolution = true;
+    private final boolean skipUsersResolution;
```

Comment:
This will stay after the feature flag is removed, so let's use the hardcoded  value:
```suggestion
    private final boolean skipUsersResolution = true;
```